llvm/lib/Passes/StandardInstrumentations.cpp

```diff
@@ -1486,6 +1486,12 @@ void VerifyInstrumentation::registerCallbacks(
                                          "\"{0}\", compilation aborted!",
                                          P));
           }
+
+          // TODO: Use complete MachineVerifierPass.
+          if (auto *MF = unwrapIR<MachineFunction>(IR)) {
+            dbgs() << "Verifying machine function " << MF->getName() << '\n';
+            verifyMachineFunction("", *MF);
```

Comment:
should we add something for `Banner`, like the "Broke module found after pass ..." above? and we have a pass that breaks MIR to test this IIRC?

Reply:
MIRs are verified immediately after parsing and there is no pass that convert valid mir to invalid mir.

Reply:
ah I was thinking of `RequireAllMachineFunctionPropertiesPass`. could you add something like that pass which creates invalid MIR, then use that for testing?

Reply:
Extended `TriggerVerifierErrorPass` now.